examples/Shim-Demo/config-overrides.js

```diff
@@ -0,0 +1,47 @@
+const { override, addWebpackPlugin, addWebpackModuleRule } = require('customize-cra');
+const AntdDayjsWebpackPlugin = require('antd-dayjs-webpack-plugin');
+const { overrideProcessEnv } = require('./misc/overrideProcessEnv');
+const esmRequire = require('esm')(module);
+const themeVariables = esmRequire('./misc/antd-theme')
+const { CompilationFeature } = esmRequire('labeling-tool/lib/enums/CompilationFeature');
+
+const ENABLED_FEATURES_NAMES = [CompilationFeature.SHOW_NOT_IMPLEMENTED];
+
+module.exports = override(
+  addWebpackPlugin(
+    new AntdDayjsWebpackPlugin({
+      replaceMoment: true,
+      plugins: require('./misc/dayjs-plugins'),
+    }),
+  ),
+  overrideProcessEnv({
+    FEATURES: JSON.stringify(ENABLED_FEATURES_NAMES),
+  }),
+  addWebpackModuleRule({
+    test: /\.svg$/i,
+    use: ['@svgr/webpack'],
```

Comment:
same thing.. we should not change build of the main application just for installing of the npm package...